xonsh/environ.py

```diff
@@ -160,6 +160,7 @@ def DEFAULT_ENSURERS():
     'XONSH_STORE_STDIN': (is_bool, to_bool, bool_to_str),
     'XONSH_TRACEBACK_LOGFILE': (is_logfile_opt, to_logfile_opt, logfile_opt_to_str),
     'XONSH_DATETIME_FORMAT': (is_string, ensure_string, ensure_string),
+    'XONSH_UPDATE_PROMPT_ON_KEYPRESS': (is_bool, to_bool, bool_to_str),
```

Comment:
Would you mind changing this to just `$UPDATE_PROMPT_ON_KEYPRESS`, and dropping the XONSH part? My concern is that that this does not effect all xonsh shells (like this name implies). Rather it just effects the ones which support it (currently only PTK). Thanks!